docs/src/components/ComponentDoc/ComponentDocAccessibility.tsx

```diff
@@ -45,29 +109,64 @@ export const ComponentDocAccessibility = ({ info }) => {
         </Text>
       )}
 
+      {((behaviorName && allAvailableBehaviors.length > 0) ||
+        (behaviorName && accIssues) ||
+        (allAvailableBehaviors.length > 0 && accIssues)) && (
+        <ul>
+          <li>
+            Behaviors
+            <ul>
+              {behaviorName && <li>{link(`Default: ${behaviorName}`, '#default-behavior')} </li>}
+              {(info.behaviors || allAvailableBehaviors.length > 0) &&
+                allAvailableBehaviors.map(variant => {
+                  return (
+                    <li>
+                      {link(
+                        `${behaviorVariantDisplayName(variant.name)}`,
+                        `#${_.kebabCase(variant.name)}`,
+                      )}
+                    </li>
+                  )
+                })}
+            </ul>
+          </li>
+          {accIssues && <li>{link('Known issues', `#${knownIsusesId}`)} </li>}
+        </ul>
+      )}
+
       {behaviorName && (
         <>
-          <Text>
-            Default behavior:{' '}
-            <a href={`behaviors/${behaviorName}#${_.kebabCase(stem)}`}>{behaviorName}</a>
-          </Text>
-          <br />
+          <Header content="Default behavior" id="default-behavior" as="h2" />
+          {behaviorMenu
+            .find(behavior => behavior.displayName === behaviorName)
+            .variations.filter(behavior => behavior.name === defaultBehaviorFileName)
+            .map(variation => (
+              <BehaviorCard variation={variation} />
+            ))}
         </>
       )}
 
-      {info.behaviors && (
+      {(info.behaviors || allAvailableBehaviors.length > 0) && (
         <>
           <Text>
-            Available behaviors:{' '}
-            {info.behaviors.map(behavior => (
-              <React.Fragment key={`${behavior.category}-${behavior.name}`}>
-                <a href={`behaviors/${behavior.category}#${_.kebabCase(behavior.name)}`}>
-                  {behavior.displayName}
-                </a>{' '}
-              </React.Fragment>
-            ))}
+            <Header content="Available behaviors" id="available-behaviors" as="h2" />
+            {allAvailableBehaviors.map(variation => {
+              return <BehaviorCard variation={variation} />
+            })}
           </Text>
-          <br />
+        </>
+      )}
+
+      {accIssues && (
+        <>
+          <Header content="Known issues" id={knownIsusesId} as="h2" />
+          <Segment className="docs-example" styles={exampleStyle}>
+            <Text style={{ whiteSpace: 'pre-line' }}>
+              <React.Suspense fallback={<Loader />}>
+                <InlineMarkdown value={accIssues} />
+              </React.Suspense>
+            </Text>
+          </Segment>
         </>
       )}
     </>
```

Comment:
I am not fully understand the styling on bottom, let's remove it all:

```diff
  return (
    <Flex column>
-      <Flex.Item>
-        <Provider styles={{ paddingLeft: '14px', background: 'transparent' }}>
          {accessibilityDetails}
-        </Provider>
-      </Flex.Item>
    </Flex>
  )
```

Reply:
removed to be consistent with other tabs